_posts/2020-10-04-newpipe-0.20.0.md

```diff
@@ -1,154 +1,176 @@
 
 ### More Translation updates and improvements
 
-Apart from the language improvements mentioned above, some other interesting localization changes happened in this release. [@comradekingu](https://github.com/comradekingu), a frequent contributor to and language checker of NewPipe and other open source projects, has once again taken a close look at our strings. There was, as always, a lot of room for improvement. Our wonderful translators rushed to help and immediately implemented his suggestions.
+On the topic of language, you may find it improved in this release, and in other forms of written NewPipe material. [Allan Nordhøy](https://github.com/comradekingu) landed another pleasant and hopefully indistingishable effort. There was, as always, unmaintained walls leaning on room for improvement. The always wonderful translators have carried those improvements onto all languages and some in turn back into English.
 
-We are also happy to provide **new localizations** in this release: Sardinian, Bengali, Portuguese (Portugal), Neapolitan and Berber.
+Happily there are **new NewPipe languages** in this release: Sardinian, Bengali, Portuguese (Portugal), Neapolitan and Berber.
 
-If you are missing a translation or think a phrase needs adjustment, head over to [Weblate](https://hosted.weblate.org/engage/newpipe/) and help us improve NewPipe's translation even more!
+Improving matters is as easy as heading over to [Weblate](https://hosted.weblate.org/engage/newpipe/) to fix even the source strings.
 
 ## Fixes
 
-If you noticed that next to offline search input now working, the actual text of the suggestions look better now, you are correct. [@TobiGr](https://github.com/TobiGr) changed the colours used to the default ones.
+Next to the now working offline search input, the actual text of the suggestions look better now. [@TobiGr](https://github.com/TobiGr) changed the colours to the NewPipe palette.
 
-An empty error report in certain kinds of crashes is not empty anymore, since [@Stypox](https://github.com/Stypox) took care of properly calling code. Yay, code! He also fixed a crash when NewPipe encounters an empty comment. Now it will be displayed just as empty as it is.
+A spooky empty error report in certain kinds of crashes will no longer be empty, courtesy of [@Stypox](https://github.com/Stypox) taking care of properly calling the code. Yay, code! He also fixed a crash when NewPipe encounters an empty comment. Now it will be displayed just as empty as it is. Ding an sich.
 
-Thanks to [@wb9688](https://github.com/wb9688), you can open a video in the local playlist tab (and potentially other locations), whereas earlier, the app would crash. And thanks to [@BoFFire](https://github.com/BoFFire), if you open the app in the Taqbaylit language, its called `Taqbaylit` now and not `Tamaziɣt Taqbaylit`.
```

Comment:
What, then who. Whereas isn't a good tool there. "And thanks to" is a terribad way to start a sentence. Never. "its" isn't "it is" or "it's".

Reply:
> What, then who.

Sure. I'm fine with either.

> Whereas isn't a good tool there. 

It is used like it is supposed to be. https://en.wiktionary.org/wiki/whereas#Conjunction

> "And thanks to" is a terribad way to start a sentence. 

Yeah, that's again something that should be a separate paragraph. Trying to join them creates problems like these.

> "its" isn't "it is" or "it's".

Correct. Slipped through.